### PR TITLE
Paint Setup: Air Powered Vertical Coaster

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

File: src/ride/coaster/air_powered_vertical_coaster.c [hunk 1]
```diff
@@ -13,3 +13,788 @@
...
+			uint32 floorImageId;
+			if (direction & 1) {
+				floorImageId = SPR_FLOOR_PLANKS_90_DEG | gTrackColours[SCHEME_SUPPORTS];
+			} else {
+				floorImageId = SPR_FLOOR_PLANKS | gTrackColours[SCHEME_SUPPORTS];
+			}
+			sub_98197C(floorImageId, 0, 0, 26, 26, 126, height, 3, 3, height, get_current_rotation());
+			sub_98199C_rotated(direction, supportsImageId, 0, 0, isDirection03 ? 26 : 26, 26, 126, height, isDirection03 ? 3 : 3, 3, height);
+		} else {
+			sub_98197C_rotated(direction, supportsImageId, 0, 0, isDirection03 ? 26 : 26, 26, 126, height, isDirection03 ? 3 : 3, 3, height);
```
Comment (janisozaur) [inline src/ride/coaster/air_powered_vertical_coaster.c]: Take it out from inside `if`, it's the same in both cases.

Comment (janisozaur) [inline src/ride/coaster/air_powered_vertical_coaster.c]: oops, `98197C` vs `_98199C`, my bad. Disregard.

File: src/ride/coaster/air_powered_vertical_coaster.c [hunk 2]
```diff
@@ -13,3 +13,788 @@
...
+	case 5:
+		if (wooden_a_supports_paint_setup(direction & 1, 0, height, gTrackColours[SCHEME_SUPPORTS], NULL)) {
+			uint32 floorImageId;
+			if (direction & 1) {
+				floorImageId = SPR_FLOOR_PLANKS_90_DEG | gTrackColours[SCHEME_SUPPORTS];
+			} else {
+				floorImageId = SPR_FLOOR_PLANKS | gTrackColours[SCHEME_SUPPORTS];
+			}
+			sub_98197C(floorImageId, 0, 0, 26, 26, 126, height, 3, 3, height, get_current_rotation());
+			sub_98199C_rotated(direction, supportsImageId, 0, 0, isDirection03 ? 26 : 26, 26, 126, height, isDirection03 ? 3 : 3, 3, height);
```
Comment (janisozaur) [inline src/ride/coaster/air_powered_vertical_coaster.c]: `isDirection03 ? 26 : 26`
`isDirection03 ? 3 : 3`
why not just `26` and `3`?

Comment (IntelOrca) [inline src/ride/coaster/air_powered_vertical_coaster.c]: Forgot to remove it

**Pull request merged**